cloud-aws/src/test/java/com/sequenceiq/cloudbreak/cloud/aws/util/AwsIamUtilTest.java

```diff
@@ -0,0 +1,286 @@
+package com.sequenceiq.cloudbreak.cloud.aws.util;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import com.amazonaws.auth.policy.Policy;
+import com.amazonaws.auth.policy.Principal;
+import com.amazonaws.auth.policy.Resource;
+import com.amazonaws.auth.policy.Statement;
+import com.amazonaws.auth.policy.Statement.Effect;
+import com.amazonaws.auth.policy.actions.S3Actions;
+import com.amazonaws.auth.policy.actions.SecurityTokenServiceActions;
+import com.amazonaws.services.identitymanagement.AmazonIdentityManagement;
+import com.amazonaws.services.identitymanagement.model.GetInstanceProfileRequest;
+import com.amazonaws.services.identitymanagement.model.GetInstanceProfileResult;
+import com.amazonaws.services.identitymanagement.model.GetRoleRequest;
+import com.amazonaws.services.identitymanagement.model.GetRoleResult;
+import com.amazonaws.services.identitymanagement.model.InstanceProfile;
+import com.amazonaws.services.identitymanagement.model.NoSuchEntityException;
+import com.amazonaws.services.identitymanagement.model.Role;
+import com.amazonaws.services.identitymanagement.model.ServiceFailureException;
+import org.junit.Assert;
```

Comment:
Could you please use JUnit 5 and assertJ (`Assertions.assertThat()`) for all new tests if possible? As I see, module `cloud-aws` lacks JUnit 5 dependencies at the moment, so feel free to add them (leaving JUnit 4 ones as well for any legacy tests). assertJ is already included in `spring-boot-starter-test`. Cloudbreak is moving away from JUnit 4, and assertJ assertions are more readable thanks to its fluent API.

Not an authoritative example, but my recent changes to `cloud-common/build.gradle` in #6700 can be simply copy-pasted. 😄